_posts/2019-07-09-10-praw-uzytecznego-designu.md

```diff
@@ -0,0 +1,124 @@
+</div>
+
+<div class="img-with-legend">
+<img alt="Portal Allegro - Oferty sponsorowane na dole listy produktów" src="/assets/img/posts/2019-07-09-design/5b.jpg" />
+<span class="img-legend">Rys. 7. Portal Allegro - Oferty sponsorowane na dole listy produktów<br />źródło: <a href="https://allegro.pl/kategoria/meble-ogrodowe-fotele-wiszace-257757">Fotele wiszące na Allegro</a> - dostęp: 2019-07-07</span>
+</div>
+
+## 6. Prawo Millera
+
+Według badań opublikowanych w 1956 roku w artykule "Magiczna liczba siedem plus minus dwa..." przez Georga A. Millera człowiek jest w stanie zapamiętać 7 informacji naraz. Dobrym przykładem wykorzystania tego prawa jest serwis Netflix. Po wejściu na stronę główną serwisu zostaniemy zasypani propozycjami filmówi i seriali, które z pewnością umilą nam niedzielny wieczór. Projektanci serwisu, zadbali o to, abyśmy nie zagineli w gąszczu propozycji, zostały one podzielone na kategorie, a każda z nich z kolei kusi 6 pozycjami.  
+
+<div class="img-with-legend">
+<img alt="Strona główna serwisu Netflix" src="/assets/img/posts/2019-07-09-design/6.jpg" />
+<span class="img-legend">Rys. 8. Strona główna serwisu Netflix<br />źródło: <a href="https://www.netflix.com/">netflix.com</a> - dostęp: 2019-07-07</span>
+</div>
+
+## 7. Prawo Teslera
+
+Stwierdza, że dla każdego systemu istnieje pewna złożoność, której nie można zmniejszyć. Zbyt duża liczba koniecznych działań może nas skutecznie zniechęcić do skorzystania z niektórych usług, jak chociażby wypełnianie mnóstwa pól formularza w sklepie internetowym podczas zakupu wymarzonej pary butów. Niestety w przypadku, gdy chcemy zarezerwować idealne miejsce na weekendowy pobyt, bez podania tych wszystkich informacji jak np. dokąd chcemy lecieć, na jak długo czy kiedy planujemy wrócić, system nie domyśli się, że chcielibyśmy spędzić tydzień w igloo na mroźnej północy Norwegii.
+
+<div class="img-with-legend">
+<img alt="Strona główna serwisu Booking.com" src="/assets/img/posts/2019-07-09-design/7.jpg" />
+<span class="img-legend">Rys. 9. Strona główna serwisu Booking.com<br />źródło: <a href="https://www.booking.com/">booking.com</a> - dostęp: 2019-07-07</span>
+</div>
+
+## 8. Prawo Hicka
+
+William Edmund Hick i Ray Hyman przeprowadzili serię eksperymentów, dzięki którym zaobserwowali, że im większa liczba opcji, tym dłużej będziemy podejmować decyzję. Oznacza to, że im więcej jest alternatywnych rozwiązań, tym więcej czasu osoba będzie potrzebowała do podjęcia decyzji, w związku z tym rozsądną strategią jest zminimalizowanie alternatyw. Jako przykład może nam posłużyć serwis Airbnb, który podszedł do problemu rezerwacji w zupełnie inny sposób niż Booking.com. Nie musimy od razu wypełniać wszystkich informacji, wystarczy, że wpiszemy miejsce do którego chcemy się udać, a resztę wypełniamy w trakcie przeglądania dostępnych ofert.
```

Comment:
"Oznacza to, że im więcej jest alternatywnych rozwiązań, tym więcej czasu osoba będzie potrzebowała do podjęcia decyzji" jest powtórzeniem poprzedniego zdania, moim zdaniem zbędnym.

Przy porównaniu Airbnb vs. Booking.com dodałbym małe "np." przed Booking.com, żeby podkreślić, że to tylko przykład, a nie jakaś wojenka tych dwóch stron.

"Nie musimy od razu wypełniać wszystkich informacji, wystarczy, że" - może zastąpić przecinek przed "wystarczy" przez myślnik dla zwiększenia czytelności?

Reply:
done